l1-contracts/test/foundry/unit/concrete/state-transition/chain-deps/facets/Mailbox/TransferEthToSharedBridge.t.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.24;
+
+import {MailboxTest} from "./_Mailbox_Shared.t.sol";
+import {IL1SharedBridge} from "contracts/bridge/interfaces/IL1SharedBridge.sol";
+
+contract MailboxTransferEthToSharedBridge is MailboxTest {
+    address baseTokenBridge;
+
+    function setUp() public virtual {
+        prepare();
+
+        baseTokenBridge = makeAddr("bridge");
+        vm.deal(diamondProxy, 1 ether);
```

Comment:
Why don't we go for fuzzed values of Ether being dealt to the diamondProxy? Something like:

```solidity
function setUp(uint256 randomAmount) public virtual {
   .
   .
   .
   vm.deal(diamondProxy, randomAmount); // This can also ensure that 0 ETH transfers are // also valid
   .
   .
   .
}
```

Reply:
Changed